source/extensions/common/wasm/wasm.cc

```diff
@@ -431,6 +431,36 @@ void continueResponseHandler(void* raw_context) {
   context->continueResponse();
 }
 
+void sendLocalResponseHandler(void* raw_context, Word response_code, Word response_code_details_ptr,
+                              Word response_code_details_size, Word body_ptr, Word body_size,
+                              Word additional_response_header_pairs_ptr,
+                              Word additional_response_header_pairs_size, Word grpc_code) {
+  auto context = WASM_CONTEXT(raw_context);
+  auto details =
+      context->wasmVm()->getMemory(response_code_details_ptr, response_code_details_size);
+  auto body = context->wasmVm()->getMemory(body_ptr, body_size);
+  auto additional_headers = toPairs(context->wasmVm()->getMemory(
+      additional_response_header_pairs_ptr, additional_response_header_pairs_size));
+  std::function<void(Http::HeaderMap & headers)> modify_headers =
```

Comment:
use auto to obtain lambda type. lambda will convert to function when it is needed

Reply:
done